react-app/src/library/services/vendor-implementations/jabra/jabra-native/jabra-native.ts

```diff
@@ -180,18 +191,18 @@ export default class JabraNativeService extends VendorImplementation {
   answerCall(): Promise<void> {
     // HACK: for some reason the headset echos an offhook event even though it was the app that answered the call rather than the headset
     this.ignoreNextOffhookEvent = true;
-    this.incomingConversationId = '';
```

Comment:
We would have `this.activeConversationId = this.incomingConversationId` or something here right? Because otherwise is `this.activeConversationId` getting set?